tests/testChebyshevBoundedField.py

```diff
@@ -113,6 +113,123 @@ def testEvaluate(self):
             self.assertFloatsAlmostEqual(scaled.evaluate(x, y), factor*z2, rtol=factor*1E-13)
             self.assertFloatsEqual(scaled.getCoefficients(), factor*field.getCoefficients())
 
+    def _testIntegrateBox(self, bbox, coeffs, expect):
+        field = lsst.afw.math.ChebyshevBoundedField(bbox, coeffs)
+        self.assertFloatsAlmostEqual(field.integrate(), expect, rtol=1E-14)
+
+    def testIntegrateTrivialBox(self):
+        """Test integrating over a "trivial" [-1,1] box.
+
+        NOTE: a "trivial" BBox can't be constructed exactly, given that Box2I
+        is inclusive, but if the [0,1] box has the same area (because it's
+        actually (-0.5, 1.5) when converted to a Box2D), and the translation
+        doesn't affect the integral.
+        """
+
+        bbox = lsst.afw.geom.Box2I(lsst.afw.geom.Point2I(0, 0), lsst.afw.geom.Point2I(1, 1))
+
+        # 0th order polynomial
+        coeffs = np.array([[5.]])
+        self._testIntegrateBox(bbox, coeffs, 4.*coeffs[0, 0])
+
+        # 1st order polynomial: odd orders drop out of integral
+        coeffs = np.array([[5., 2.], [3., 4.]])
+        self._testIntegrateBox(bbox, coeffs, 4.*coeffs[0, 0])
+
+        # 2nd order polynomial in x, 0th in y
+        coeffs = np.array([[5., 0., 7.]])
+        self._testIntegrateBox(bbox, coeffs, 4.*coeffs[0, 0] - 4./3.*coeffs[0, 2])
+
+        # 2nd order polynomial in y, 0th in x
+        coeffs = np.zeros((3, 3))
+        coeffs[0, 0] = 5.
+        coeffs[2, 0] = 7.
+        self._testIntegrateBox(bbox, coeffs, 4.*coeffs[0, 0] - 4./3.*coeffs[2, 0])
+
+        # 2nd order polynomial in x and y, no cross-term
+        coeffs = np.zeros((3, 3))
+        coeffs[0, 0] = 5.
+        coeffs[1, 0] = 7.
+        coeffs[0, 2] = 3.
+        self._testIntegrateBox(bbox, coeffs, 4.*coeffs[0, 0] - 4./3.*coeffs[2, 0] - 4./3.*coeffs[0, 2])
+
+    def testIntegrateBox(self):
+        """Test integrating over an "interesting" box.
+
+        The values of these integrals were checked in Mathematica. The code
+        block below can be pasted into Mathematica to re-do those calculations.
+
+        .. code:: mathematica
```

Comment:
This may not work as I don't see `mathematica` listed on Pygment's supported languages. You could just do a general pre-formatted block:

```
.. code:: mathematica
```

to

```
::
```

On the other hand, we're not planning rendering unit test docstrings in Sphinx, so this isn't a significant point.